concepticondata/conceptlists.tsv

```diff
@@ -419,4 +419,5 @@ Wilson-2015-173	Wilson, Fiona M.	2015		173	basic, ranked, norms	Latin	Spanish		W
 Chacon-2022-94	Chacon, Thiago C.	2022		94	basic	English,Portuguese	Northwest Arawakan Languages		Chacon2022		This concept list is taken from a dataset on Northwest Arawakan Languages. The list is based on [Swadesh's list of 100 items](:ref:Swadesh-1955-100).		
 Lionnet-1985-364	Lionnet, André	1985		364	basic	Spanish	Uto-Aztecan Languages		Lionnet1985		This concept list is taken from a dataset on Uto-Aztecan languages. The study sets to demonstrate the existence of a Sonorense branch inside the linguistic family and proposes an inner classification of the branch. The data was collected by André Lionnet.	25-57	
 Oliveira-2014-517	Oliveira, Sanderson Castro Soares de	2022		517	basic	English	Panoan languages	https://repositorio.unb.br/handle/10482/17129	Oliveira2014	Oliveira2014	This list presents the concepts that are reconstructed for Proto-Panoan. In the original data, the reflexes are presented for various Panoan languages.	397-491	
-Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022		348	basic	French	English		Hantgan2022		This concept list is part of a study on the influence on Bangime by neighboring languages.		
\ No newline at end of file
+Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022		348	basic	French	English		Hantgan2022		This concept list is part of a study on the influence on Bangime by neighboring languages.		
+Gravina-2014-1611	Gravina, Richard	2014		1611	basic	English,French	Central Chadic languages	https://scholarlypublications.universiteitleiden.nl/handle/1887/30139	Gravina2014		This concept list is taken from a dataset that on Central Chadic languages. The dataset accompanies a PhD thesis that looks at the diverse phonological systems found within Central Chadic, and reconstructs the phonological system of their ancestor language. The study also describes the phonological processes that led from this system to the many phonological systems that are in use today, shedding light both on the history of the languages, and on issues in the analysis of these languages.		
```

Comment:
French does not seem to be a source language, as it is given as "Gloss". The source is probably only English.

In the description, there is a small typo/error: "that on"

Reply:
Or rather: Are we sure that "Gloss" in the original file actually means what it should mean? Or is indeed the French the original, and English is the Gloss? This needs to be clarified.